docs/nginx/monolith-sample.conf

```diff
@@ -0,0 +1,24 @@
+server {
+    listen 443 ssl;
+    server_name my.hostname.com;
+
+    ssl_certificate /path/to/fullchain.pem;
+    ssl_certificate_key /path/to/privkey.pem;
+    ssl_dhparam /path/to/ssl-dhparams.pem;
+
+    proxy_set_header Host      $host;
+    proxy_set_header X-Real-IP $remote_addr;
```

Comment:
Do we use `X-Real-IP` anywhere?

Reply:
Don't think so, but figured it might as well be included in case we ever want to add IP filtering.

Reply:
I think the general header is `X-Forwarded-For` - at least it is on Apache.